docs/website/docs/command-reference/deploy.md

```diff
@@ -9,11 +9,12 @@ necessary to deploy the components.
 When running the command `odo deploy`, `odo` searches for the default command of kind `deploy` in the devfile, and executes this command.
 The kind `deploy` is supported by the devfile format starting from version 2.2.0.
 
-The `deploy` command is typically a *composite* command, composed of several *apply* commands:
-- a command referencing an `image` component that, when applied, will build the image of the container to deploy, and push it to its registry,
-- a command referencing a [`kubernetes` component](https://devfile.io/docs/2.2.0/defining-kubernetes-resources) that, when applied, will create a Kubernetes resource in the cluster.
+The `deploy` command is typically a *composite* command, composed of several *apply* and *exec* commands:
+- an `apply` command referencing an `image` component that, when applied, will build the image of the container to deploy, and push it to its registry,
+- an `apply` command referencing a [`kubernetes` component](https://devfile.io/docs/2.2.0/defining-kubernetes-resources) that, when applied, will create a Kubernetes resource in the cluster.
+- an `exec` command referencing a container component that, when applied, will run the command defined by `commandLine` inside a container started by a Kubernetes Job; read more about it [here](../../development/devfile#how-odo-runs-exec-commands-in-deploy-mode).
```

Comment:
🤔 It's weird - the build is failing on Netlify due to a broken link : https://app.netlify.com/sites/odo-docusaurus-preview/deploys/641d419c585fe20008b8c41a
But with the previous change, my browser could not resolve the link:

[Screencast from 2023-03-24 09-19-03.webm](https://user-images.githubusercontent.com/593208/227464067-39dbda28-935b-40aa-b8bd-4c85be5d4989.webm)

Do you have similar issues with this page: https://deploy-preview-6672--odo-docusaurus-preview.netlify.app/docs/command-reference/deploy/

Reply:
I am reverting the previous commit because it was working fine for me, at least locally. I am going to test it one more time here.